### DISCUSSION TITLE
Always get 204 no content when using async await on lucid orm

### DISCUSSION THREAD

**Discussion by DeVoresyah** [adonisjs/core#1472] [Help]
version:
```
"@adonisjs/ace": "^5.0.8",
"@adonisjs/auth": "^3.0.7",
"@adonisjs/bodyparser": "^2.0.5",
"@adonisjs/cors": "^1.0.7",
"@adonisjs/fold": "^4.0.9",
"@adonisjs/framework": "^5.0.9",
"@adonisjs/ignitor": "^2.0.8",
"@adonisjs/lucid": "^6.1.3",
"adonis-fold": "^3.0.3",
```

Example controller:
```
async index ({ request, response }) {
const user = await User.all()
return response.status(200).json({ data: user })
}
```
it always return 204 no content. But when I comment `const user = await User.all()` and return a dummy data, it's fine

```
async index ({ request, response }) {
// const user = await User.all()
return response.status(200).json({ data: "Hello world~!" })
}
```

**Comment by thetutlage** [comment]
What happens when you `console.log(user)` before returning it in the response?

**Reply by DeVoresyah** [reply]
it returns the data (which is I expected)

**Reply by DeVoresyah** [reply]
I found something interesting here, when I add `response.implicitEnd = false` in the first line on middleware. the response work well

**Reply by thetutlage** [reply]
That shouldn't be the case though. I suggest removing the `return` keyword and then give it a try. If the issue still persists, then please create a sample repo with the required code to debug the issue.

```ts
async index ({ request, response }) {
  const user = await User.all()
  response.status(200).json({ data: user })
}
```
